docs/writing-rules/rule-syntax.md

```diff
@@ -759,6 +760,74 @@ int("2147483646")
 
 This removes quotes (`'`, `"`, and `` ` ``) from both ends of the metavariable content. As a result, Semgrep detects `"2147483648"`, but it does **not** detect `"2147483646"`. This is useful when you expect strings to contain integer or float data.
 
+### `metavariable-name`
+
+The `metavariable-name` operator adds a constraint to the types of identifiers a metavariable is able to match. Currently the only constraint supported is on module or namespace an identifier originates from. This is useful for filtering results in languages which don't have a native syntax for fully qualified names, or languages where module names may contain characters which are not legal in identifiers (e.g., JavaScript, TypeScript). 
```

Comment:
```suggestion
The `metavariable-name` operator adds a constraint to the types of identifiers a metavariable is able to match. Currently the only constraint supported is on the **module** or **namespace** an identifier originates from. This is useful for filtering results in languages which don't have a native syntax for fully qualified names, or languages where module names may contain characters which are not legal in identifiers, such as JavaScript or TypeScript). 
```

Reply:
@kopecs This is the only remaining comment to resolve - are you ok with committing the suggestion?

Reply:
Bold/italics on module or namespace seem a bit gratuitous to me, but adding "the" and rewording to "such as" instead of "e.g." sounds good.